CSCD488-Winter2024/senior-project-the-a-team
# can now view the posts you've made

- A user can now view the posts they've made
- A Viewer can edit the posts they've made in my posts page

To test:
- ensure that every kind of post appears in my posts
- ensure that a viewer account can edit their post in my posts
- make sure my post page is functional as in:
        - it disables when navigating to another page
        - every other page disables when navigating to my post page
        - title updates as expected
